cranelift-codegen/meta/src/isa/x86/mod.rs

```diff
@@ -32,13 +32,12 @@ pub(crate) fn define(shared_defs: &mut SharedDefinitions) -> TargetIsa {
     let mut x86_32 = CpuMode::new("I32");
 
     let expand_flags = shared_defs.transform_groups.by_name("expand_flags");
-    let narrow_flags = shared_defs.transform_groups.by_name("narrow_flags");
-    let widen = shared_defs.transform_groups.by_name("widen");
+    let widen = shared_defs.transform_groups.by_name("x86_widen");
```

Comment:
Perhaps we should rename this to `x86_widen` (here and where used) to be very clear what is going on.

Reply:
Oh yeah I can definitely do that!

Reply:
Sounds good, I'll merge this after your push.